src/client/utils/event.js

```diff
@@ -53,7 +53,7 @@ export function stopPropagation (ev) {
 }
 
 export function isObjectEventListener (listener) {
-    return typeof listener === 'object' && typeof listener.handleEvent === 'function';
+    return listener && typeof listener === 'object' && typeof listener.handleEvent === 'function';
```

Comment:
First check needs only for preventing error on access to the `listener.handleEvent` property then `listener` is null.
Let's rephrase as 
`return typeof listener === 'object' && typeof (listener && listener.handleEvent) === 'function'`;

Reply:
Then `return typeof listener === 'object' && listener && listener.handleEvent === 'function';`

Reply:
I don't understand what do you mean

Reply:
Sorry I am confused, that's right variant:
`return typeof listener === 'object' && listener && typeof listener.handleEvent === 'function';`

Reply:
Ok